internal/service/streaminstance/resource_stream_instance.go

```diff
@@ -83,6 +92,16 @@ func (r *streamInstanceRS) Schema(ctx context.Context, req resource.SchemaReques
 				ElementType: types.StringType,
 				Computed:    true,
 			},
+			"stream_config": schema.SingleNestedAttribute{
+				Optional: true,
+				Computed: true,
+				Attributes: map[string]schema.Attribute{
+					"tier": schema.StringAttribute{
```

Comment:
we would need to adjust our documentation for the resource and data sources

Reply:
also, the example should be updated

Reply:
Done! Thanks